src/syntax_mapping.rs

```diff
@@ -26,14 +26,15 @@ pub enum MappingTarget<'a> {
 #[derive(Debug, Clone, Default)]
 pub struct SyntaxMapping<'a> {
     mappings: Vec<(GlobMatcher, MappingTarget<'a>)>,
+    pub ignored_suffixes: IgnoredSuffixes,
```

Comment:
Make this field private, which makes the public API much more future proof.

Reply:
I couldn't make it fully private since it's used in `src/assets.rs`.
So I changed it to `pub(crate)`. Let me know please if you have a better solution

Reply:
That works great, the important part is that it stopped being public API 👍